doc/overview.rst

```diff
@@ -573,6 +573,443 @@ Here is an example:
+SageMaker models. JumpStart relies on Docker images from the
+following repos https://github.com/aws/deep-learning-containers,
+https://github.com/aws/sagemaker-xgboost-container,
+and https://github.com/aws/sagemaker-scikit-learn-container. Use
+the ``model_id`` and ``version`` of the corresponding model to
+retrieve the related image as follows.
+
+.. code:: python
+
+   from sagemaker import image_uris
+
+   image_uri = image_uris.retrieve(
+       region=None,
+       framework=None,
+       image_scope=scope,
+       model_id=model_id,
+       model_version=model_version,
+       instance_type="ml.m5.xlarge",
+   )
+
+Deploy a  Pre-Trained Model Directly to a SageMaker Endpoint
+============================================================
+
+In this section, you learn how to take a pre-trained JumpStart model
+and deploy it directly to a SageMaker Endpoint. This is the fastest
+way to start machine learning with a JumpStart model. The following
+assumes familiarity with `SageMaker
+models <https://sagemaker.readthedocs.io/en/stable/api/inference/model.html>`__
+and their deploy functions.
+
+To begin, select a ``model_id`` and ``version`` from the JumpStart
+models table, as well as a model scope of either “inference” or
+“training”. For this example, you use a pre-trained JumpStart model,
+so select “inference”  for your model scope. Use the utility
+functions to retrieve the URI of each of the three components you
+need to continue.
+
+.. code:: python
+
+   from sagemaker import image_uris, model_uris, script_uris
+
+   model_id, model_version = "tensorflow-tc-bert-en-cased-L-12-H-768-A-12-2", "1.0.0"
+   instance_type, instance_count = "ml.m5.xlarge", 1
+
+   # Retrieve the URIs of the JumpStart resources
+   base_model_uri = model_uris.retrieve(
+       model_id=model_id, model_version=model_version, model_scope="inference"
+   )
+   script_uri = script_uris.retrieve(
+       model_id=model_id, model_version=model_version, script_scope="inference"
+   )
+   image_uri = image_uris.retrieve(
+       region=None,
+       framework=None,
+       image_scope="inference",
+       model_id=model_id,
+       model_version=model_version,
+       instance_type=instance_type,
+   )
+
+Next, pass the URIs and other key parameters as part of a new
+SageMaker Model class. The ``entry_point`` is a JumpStart script
+named ``inference.py``. JumpStart handles the implementation of this
+script. You must use this value for model inference to be successful.
+For more information about the Model class and its parameters,
+see `Model <https://sagemaker.readthedocs.io/en/stable/api/inference/model.html>`__.
+
+.. code:: python
+
+   from sagemaker.model import Model
+   from sagemaker.session import Session
+
+   # Create the SageMaker model instance
+   model = Model(
+       image_uri=image_uri,
+       model_data=base_model_uri,
+       source_dir=script_uri,
+       entry_point="inference.py",
+       role=Session().get_caller_identity_arn(),
```

Comment:
future PR: All JumpStart models can be deployed in network isolation. How about passing the network isolation argument set to `True` here?

Reply:
We don't make reference to network isolation in the P0 docs. I agree that there should be some documentation for this in the future.